data/motd/meson.build

```diff
@@ -54,3 +56,16 @@ if libsystemd.found() and get_option('man')
     install_dir: join_paths(mandir, 'man8'),
   )
 endif
+
+if libsystemd.found() and get_option('systemd_unit_user') != ''
+  con3 = configuration_data()
+  con3.set('localstatedir', localstatedir)
+  con3.set('systemd_unit_user', get_option('systemd_unit_user'))
+  configure_file(
+    input: 'fwupd.conf',
+    output: 'fwupd.conf',
+    configuration: con3,
+    install: true,
+    install_dir: systemd_sysusers_dir,
```

Comment:
Does any systemd trigger need to be called after this file is added or does it keep an inotify?

Any special thing needed to be done when uninstalling l?

Reply:
According to https://www.freedesktop.org/software/systemd/man/sysusers.d.html it happens magically -- "_at package installation or boot time_".